### ISSUE TITLE
Attempt handling Enter key for Grid and ListView to open instead of edit

### ISSUE THREAD

**Issue by Iankodj** [telerik/kendo-ui-core#5546]
Currently, the Grid and the ListView utilize keyboard shortcuts that cannot be changed. 

In the FileManager Enter key is expected to Open folders/files instead of renaming it.

**Comment by GaloisGirl** [comment]
This is related to accessibility, since at the moment, there is no way to open files via keyboard.

**Comment by Dimitar-Goshev** [comment]
After the enhancement there is a JS error thrown in Grid View in the following scenario:
1. Go to Kendo UI FileManager Index demo on staging.
1. Click on New Folder button.
1. Switch to Grid View.
1. Click on Breadcrumb root icon - the icon is focused.
1. Press Tab to move focus to Grid View.
1. Navigate to the New Folder with arrow keys.
1. When the New folder is focused, press Enter - New folder is loaded in the Grid View
1. Press Down Arroy Key.

Current - `Cannot read property 'closest' of null at init._kendoKeydown`
Expected - no error
NOTE: The issue is not reproducible with the default folders Documents and Images on Kendo UI demo. But the issue is reproducible even with them on MVC index demo.

**Comment by Dimitar-Goshev** [comment]
- [ ] The error above is fixed. But now if on step 8, instead of Down Arrow, we press Enter, the following error is thrown:
Cannot read property 'is' of null at init._triggerOpen
